FlightJournal.Web/FlightJournal.Web.csproj

```diff
@@ -1190,7 +1190,7 @@
     <Content Include="Views\Report\_Logbook.cshtml" />
     <Content Include="Views\Shared\DisplayTemplates\LandingCount.cshtml" />
     <Content Include="Views\TrainingStatus\PilotStatusDetails.cshtml" />
-    <Content Include="Views\TrainingLog\SelectTrainingProgram.cshtml" />
+    <Content Include="Views\Flight\PartialGrid.cshtml" />
```

Comment:
did you mean to remove SelectTrainingProgram in this step ?

Reply:
Looks like a merge error